src/lua/init.lua

```diff
@@ -124,431 +126,474 @@ local function pathJoin(...)
   return path
 end
 
-return function(args)
+-- Bundle from folder on disk
+local function folderBundle(base)
+  local bundle = { base = base }
 
-  local uv = require('uv')
-  local luvi = require('luvi')
-  local env = require('env')
+  function bundle.stat(path)
+    path = pathJoin(base, "./" .. path)
```

Comment:
no, because luvi's path join will ignore the base if path begins with a slash (sortof works like path.resolve).  This is to force path to be relative to base.